pkg/controller/v1beta1/inferenceservice/controller.go

```diff
@@ -95,193 +94,215 @@ type InferenceServiceReconciler struct {
 func (r *InferenceServiceReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
 	_ = context.Background()
 
-	// Fetch the InferenceService instance
 	isvc := &v1beta1api.InferenceService{}
-	if err := r.Get(ctx, req.NamespacedName, isvc); err != nil {
-		if apierr.IsNotFound(err) {
-			// Object not found, return.  Created objects are automatically garbage collected.
-			// For additional cleanup logic use finalizers.
-			return reconcile.Result{}, nil
+	var result reconcile.Result
+	err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
```

Comment:
it is better to wrap `retry.RetryOnConflict` on `Reconcile` instead if the entire function needs to be in the retry block.